_contributors-app/shivank01.md

```diff
@@ -0,0 +1,113 @@
+<<<<<<< HEAD
+---
+name: Shivank Pathak
+description: Pythonista, Machine Learning/Deep Learning Enthusiast, Competitive Programmer
+picUrl: https://avatars2.githubusercontent.com/u/22510918?s=400&v=4
+skills: |
+  - **Languages:** Python, C/C++
+  - **Frameworks:** Keras, Tensorflow, scikit-learn, nltk, opencv
+  - **Tools:** git, Github, Latex
+
+githubUsername: shivank01
+speciality: |
+
+  - Machine Learning/Deep Learning
+
+  - Data Structures and Algorithms
+
+  - Natural Language Processing
+
+  - Computer Vision
+
+github_url: https://github.com/shivank01
+linkedIn_url: https://www.linkedin.com/in/shivank01/
+twitter_url: https://twitter.com/shivank__
+
+work_experience: |
+ - Research Intern @[Indian Institute of Technology, BHU](https://www.iitbhu.ac.in/dept/cse/research/specializations/specialization-2): Currently working on Research Paper Collaborator Recommendation System using Deep Learning and Natural Language Processing [May 2018 - Present]
+
+project:
+- |
+    - [Twitter Sentiment Analysis](https://github.com/shivank01/NaturalLanguageProcessing): Live sentiment analysis of twitter.Draws the sentiment graphs of any topic of twitter.
+- |
+
+   - [Handwritten Digit Recognition](https://github.com/shivank01/DeepLearning): Recognizes the handwritten digits using Deep Learning on MNIST dataset.
+
+- |
+   - [Employee Tracking System](https://github.com/shivank01/Employee-tracking-system): Tracks the employee in real time. It is the Hybrid Android Application made using Apache Cordova.
+
+- |
+   - [Hibiscut](http://www.github.com/shivank01/web-crawler-and-scraper/blob/master/HIBISCUT.py): Automatically sends the official notices to the students via sms from the online notice board ’Hibiscus’ of IIIT Bhubaneswar.
+
+volunteer_experience: |
+ -  I like implementing Machine Learning algorithms and Deep Learning in real world. I also like Competitive Programming.
+
+ - I am Secretary of the Association for Computing Machinery(ACM) IIIT-bh Student Chapter.
+
+ - I am a member of the Programming Society of IIIT-Bh. Together we try to guide the first years in getting started with software development and aim to build the coding culture of our college.
+
+ - I've successfully organized a two-day bootcamp on a basic primer to python and Machine Learning.
+
+ - I am also the Web Administrator of Online Mess Portal 'Rosei' of IIIT Bhubaneswar of which I manage Code, database and server.
+
+ - Finalist in the team based GDG devfest hackathon. Made a review analysis of products on e-commerce websites using Machine Learning.
+
+ - 4th Runner's up in the pan India Competitive coding contest Kronothon with over 7000 participants.
+---
+||||||| merged common ancestors
```

Comment:
Another conflict marker

Reply:
ok Will correct it and make it under one commit